### DISCUSSION TITLE
Card component: Under page title or above page title?

### DISCUSSION THREAD

**Discussion by bhoomasri-ux** [twilio-labs/paste#3374] [General] [answered]
We are using a card component to introduce and potentially redirect the user to link insights page (attached image). Is there a recommendation on where the card should be place? Below the title of the page or above?
![image](https://github.com/twilio-labs/paste/assets/82901489/7dc3bef6-73bc-4767-8332-a52e66487500)

**Comment by serifluous** [comment, marked as answer]
Hey @bhoomasri-ux ! For placement, I'd recommend thinking about it along the lines of: If you were to read out the text on this page to a customer, when would it make sense to introduce them to Link Insights? Do they need to read about link shortening first, before you introduce them to link insights? Or are people on this page already going to be so familiar with link shortening, that you can introduce them to link insights right away?

One thing you could try out, too, is using Callout for the Link Insights card instead, especially if the intention is to get folks to pay a little more attention to the feature: https://paste.twilio.design/components/callout

Let us know what you decide to do!
